src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -4,8 +4,12 @@ include_directories(../unixcoreruncommon)
 
 add_compile_options(-fPIE)
 
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+include(../unixcoreruncommon/configure.cmake)
```

Comment:
You could include the configure.cmake in the parent folder's CMakeLists.txt instead. That way, cmake would need to run the configure script just once. Basically, you'd add this to the parent CMakeLists.txt to the `if` for Unix:
```cmake
include_directories(${CMAKE_CURRENT_BINARY_DIR})
include(unixcoreruncommon/configure.cmake)
```
And remove the following from all the three files:
```cmake
set(CMAKE_INCLUDE_CURRENT_DIR ON)
include(../unixcoreruncommon/configure.cmake)
```

Reply:
Yay, fixed.